javascript/ql/src/Security/CWE-078/UnsafeShellCommandConstruction.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Unsafe shell command constructed from library input
+ * @description Using externally controlled strings in a command line may allow a malicious
+ *              user to change the meaning of the command.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id js/shell-command-constructed-from-input
+ * @tags correctness
+ *       security
+ *       external/cwe/cwe-078
```

Comment:
Not sure if there is a convention in the naming here and if we need the zeros in the CWE numbering in the tags section:
The MITRE links are as follows:
https://cwe.mitre.org/data/definitions/78.html
https://cwe.mitre.org/data/definitions/88.html